src/typings.d.ts

```diff
@@ -2,7 +2,7 @@
 // https://github.com/typings/typings
 // https://www.typescriptlang.org/docs/handbook/writing-declaration-files.html
 
-declare var module: { id: string };
-declare var require: any;
-declare var TweenMax: any;
-declare var {}: any;
+// declare var module: { id: string };
```

Comment:
why remove this?

Reply:
I don't see why this is necessary, these types are provided else ware. It was throwing a duplicate identifier error.